client/my-sites/activity/filterbar/type-selector/activity-type-selector.jsx

```diff
@@ -12,7 +12,13 @@ import { TypeSelector } from './type-selector';
 const ActivityTypeSelector = ( props ) => {
 	const { translate } = props;
 
-	return <TypeSelector { ...props } title={ translate( 'Activity Type' ) } />;
+	return (
+		<TypeSelector
+			{ ...props }
+			title={ translate( 'Activity Type' ) }
+			showAppliedFiltersCount={ true }
```

Comment:
`ActivityTypeSelector` is only used by the Activity Log, so we need to default it here as `true`, given that `TypeSelector` is also [used by `IssueTypeSelector` on Jetpack Manage](https://github.com/Automattic/wp-calypso//blob/16453d9100df016ac8e3d1c1d5799d6f5b8b01ad/client/my-sites/activity/filterbar/type-selector/issue-type-selector.tsx#L13).